drizzlepac/adrizzle.py

```diff
@@ -73,11 +71,8 @@ def drizzle(input, outdata, wcsmap=None, editpars=False, configObj=None, **input
         run(configObj, wcsmap=wcsmap)
 
 
-#
-# ###  User level interface to run drizzle tasks from TEAL
```

Comment:
Same comment as previous modules.  Either leave the comment and add more doc indicating TEAL GUI being removed, but some functionality will remain as a convenience to reading default parameters from where?